splunk/scripts/log_indexing.sh

```diff
@@ -5,9 +5,25 @@ endpoint="https://localhost:8089/services/data/inputs/oneshot"
 SPLUNK_PASSWORD="Password"
 
 echo "Indexing logs..."
+max_retries=3
+
 for file in "$logs_path"/*; do
   echo "Indexing $file"
-  curl  --insecure --user admin:"$SPLUNK_PASSWORD" "$endpoint" --data name="$file" --data index="test_index"
+  retries=0
+
+  while [ $retries -lt $max_retries ]; do
+    if curl --insecure --user admin:"$SPLUNK_PASSWORD" "$endpoint" --data name="$file" --data index="test_index"; then
+      break
+    else
+      echo "Encountered an error while initiating indexing for $file. Retrying..."
+      retries=$((retries + 1))
+    fi
+
+    if [ $retries -eq $max_retries ]; then
```

Comment:
can we somehow have more output here to help figuring out what happened?

Reply:
stderr is printed to the console\log anyway, do we need more\duplicate output?

Reply:
in your experience, was that sufficient to debug issues?

Reply:
```
Indexing logs...
Indexing /opt/splunk/test_logs/log_events.log
(7) Failed to connect to localhost port 8085: Connection refused
Encountered an error while initiating indexing for /opt/splunk/test_logs/log_events.log. Retrying...
```

Seems pretty sufficient to me... Is there anything you think should be added?